<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>335</width>
    <height>258</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Adicionar</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="leAutor">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>100</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbAno">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>130</y>
      <width>47</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Ano</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbQuantidade">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>160</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Quantidade</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leAno">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>130</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbAutor">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>100</y>
      <width>47</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Autor</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbAdicionarAtualizar">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>30</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Adicionar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbInserirAtualizar">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>200</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Inserir</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbNome">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>47</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Nome</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leNome">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>70</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="leQuantidade">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>160</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>leNome</tabstop>
  <tabstop>leAutor</tabstop>
  <tabstop>leAno</tabstop>
  <tabstop>leQuantidade</tabstop>
  <tabstop>pbInserirAtualizar</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
